frontend/src/components/auth/ForgotPassword.tsx

```diff
@@ -0,0 +1,110 @@
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  Center,
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  Image,
+  Input,
+  Stack,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import authAPIClient from "../../APIClients/AuthAPIClient";
+/* Images */
+import CCBCLogo from "../../images/ccbc-logo.png";
+import LoginGraphic from "../../images/Login-graphic.png";
+import ForgotPasswordEmailModal from "./ForgotPasswordEmailModal";
+
+const ForgotPassword = (): React.ReactElement => {
```

Comment:
nit: add documentation for this component

Reply:
What does documentation for a React component look like? I don't think we are documenting components in the project and starter code doesn't have any examples either. Could you link an example?

Reply:
https://github.com/uwblueprint/ccbc/pull/48/files